<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ink</class>
 <widget class="QDialog" name="ink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ink</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>喷印参数</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>501</width>
         <height>401</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>语言文本</string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>301</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>30</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>添加文本</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>计数文本</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>72</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>数量</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>210</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>添加文本</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>321</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>331</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>150</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>150</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>位数</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>60</y>
           <width>93</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>上升序列</string>
         </property>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>331</width>
           <height>241</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_7">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QCheckBox" name="checkBox_18">
            <property name="text">
             <string>H</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_9">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QCheckBox" name="checkBox_6">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_8">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBox_16">
            <property name="text">
             <string>F</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="checkBox_17">
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_14">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBox_26">
            <property name="text">
             <string>P</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QCheckBox" name="checkBox_23">
            <property name="text">
             <string>M</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QCheckBox" name="checkBox_29">
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox_10">
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_19">
            <property name="text">
             <string>I</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QCheckBox" name="checkBox_24">
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBox_21">
            <property name="text">
             <string>K</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_20">
            <property name="text">
             <string>J</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="checkBox_27">
            <property name="text">
             <string>Q</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_15">
            <property name="text">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QCheckBox" name="checkBox_30">
            <property name="text">
             <string>T</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_25">
            <property name="text">
             <string>O</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QCheckBox" name="checkBox_28">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="checkBox_22">
            <property name="text">
             <string>L</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QCheckBox" name="checkBox_11">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_13">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QCheckBox" name="checkBox_12">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBox_31">
            <property name="text">
             <string>U</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkBox_32">
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QCheckBox" name="checkBox_33">
            <property name="text">
             <string>W</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QCheckBox" name="checkBox_34">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QCheckBox" name="checkBox_35">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QCheckBox" name="checkBox_36">
            <property name="text">
             <string>Z</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>320</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>步长</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>320</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>时间文本</string>
        </attribute>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>MM.dd</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>yyyy.MM.dd</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>64</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>yy.MM.dd</string>
         </property>
        </widget>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>时间文本</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>110</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="cursorPosition">
          <number>0</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>预览效果</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>150</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>添加文本</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>430</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>预览打印信息</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>540</y>
         <width>641</width>
         <height>51</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>550</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>610</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>261</width>
         <height>431</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>信息1</string>
        </attribute>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>401</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>信息2</string>
        </attribute>
        <widget class="QListWidget" name="listWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>401</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>信息3</string>
        </attribute>
        <widget class="QListWidget" name="listWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>401</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>信息4</string>
        </attribute>
        <widget class="QListWidget" name="listWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>401</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_9">
        <attribute name="title">
         <string>信息5</string>
        </attribute>
        <widget class="QListWidget" name="listWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>401</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="printmark.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ink</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>674</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>740</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ink</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>669</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>718</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
